### PR TITLE
Update windows-update-issues-troubleshooting.md

### PR DESCRIPTION
**Pull request by kartv-ms**
Added a section for transient errors where users do not have to take remedial action.

### CODE REVIEW

File: support/windows-client/deployment/windows-update-issues-troubleshooting.md [hunk 1]
```diff
@@ -271,3 +271,14 @@ Other components that connect to the internet:
...
+## Transient errors caused by heavy load or network congestion
+
+Users might sometimes receive the following errors from Windows Update. These are transient issues, occurring when the service is temporarily under heavy load or when networks are congested. Users do not have to take any action because the device will retry the operation at a later time. 
+
+|Error Code |Error Value | Details
+|---------|---------|---------|
+|WU_S_SEARCHT_LOAD_SHEDDING | 0x248001 | Search operation completed successfully but one or more services were shedding load.
+|WU_E_PT_LOAD_SHEDDING | 0x8024402d | The server is shedding load.
+
+In the above cases, users programatically calling into the Windows Update Agent API to [retreive the result](https://learn.microsoft.com/windows/win32/api/wuapi/nf-wuapi-isearchresult-get_resultcode) of a search operation would get [orcFailed](https://learn.microsoft.com/windows/win32/api/wuapi/ne-wuapi-operationresultcode) or [orcSucceededWithErrors](https://learn.microsoft.com/windows/win32/api/wuapi/ne-wuapi-operationresultcode). Retrying the operation at a later time is expected to succeed.
```
Comment (gewarren) [inline support/windows-client/deployment/windows-update-issues-troubleshooting.md:284]:
```suggestion
In these cases, users that programmatically call into the Windows Update Agent API to [retrieve the result](/windows/win32/api/wuapi/nf-wuapi-isearchresult-get_resultcode) of a search operation would get [orcFailed](/windows/win32/api/wuapi/ne-wuapi-operationresultcode) or [orcSucceededWithErrors](/windows/win32/api/wuapi/ne-wuapi-operationresultcode). Retrying the operation at a later time is expected to succeed.
```

**Comment by kartv-ms** [comment]
#sign-off

**Comment by aczechowski** [comment]
I'll signoff, but want to inform @Deland-Han (probably should fix metadata on this article at some point 😄 )

#sign-off

**Comment by Deland-Han** [comment]
We created a bug to triage this change request. We will response soon.

**Comment by v-lianna** [comment]
@kartv-ms The changes are live at https://learn.microsoft.com/en-us/troubleshoot/windows-client/deployment/windows-update-issues-troubleshooting. This PR will be closed.

**Pull request closed without merging**